x/auth/vesting/msg_server.go

```diff
@@ -160,5 +184,180 @@ func (s msgServer) CreatePeriodicVestingAccount(goCtx context.Context, msg *type
 		),
 	)
 	return &types.MsgCreatePeriodicVestingAccountResponse{}, nil
+}
+
+func max64(a, b int64) int64 {
```

Comment:
isn't this a dup from the function in types/?

Reply:
Yes, and IMHO it's a trivial enough function that it's fine to define locally where needed. E.g. the standard library defines it for `int` but doesn't bother to export it.